docs/advanced/debugging.rst

```diff
@@ -3,16 +3,36 @@ Debugging
 =========
 
 Logging
-=======
+-------
 
-The Flocker processes running on the nodes will write their logs to ``/var/log/flocker/``.
-The log files are named ``<processname>-<pid>.log``, e.g. ``flocker-volume-1234.log``.
+Flocker processes generally use `eliot`_ for logging.
+These logs can be rendered as an ASCII tree using `eliottree`_.
```

Comment:
It would be really good, perhaps as a follow-up to have tools we've tested with Flocker logs (elasticsearch?) and tips on how to use them.